apps/website/src/pages/_global.astro

```diff
@@ -10,6 +10,8 @@ import '@fontsource/noto-sans/600.css';
 import '@fontsource/noto-sans/300.css';
 import '@fontsource/noto-sans/800.css';
 
+import '@docsearch/css';
```

Comment:
if you import this inside a style tag, you will be able to use cascade layers like this:

```css
@import '@docsearch/css' layer algolia;
```

this will avoid the need for `!important` when overriding variables

Reply:
Thanks, I am now using `body` and I don't need `!important` anymore. The Algolia styles still appear outside the cascade layer, but the `<style>` that contains `body` (instead of `:root`) appears higher in the layer specificity list. Just wondering, why could that be the case?

Here is the screenshot where the first layer is the Algolia class, the second layer with `body` are my classes, the third layer with `:root` is the duplicate layer, and the fourth layer are the algolia styles inside the cascade layer.

![image](https://github.com/iTwin/iTwinUI/assets/45748283/03adce7e-7a79-4b26-b309-1542b93b76fc)
![image](https://github.com/iTwin/iTwinUI/assets/45748283/ba4f19d7-cc30-4975-82e3-bcda8adea6c5)
<!-- ![image](https://github.com/iTwin/iTwinUI/assets/45748283/db58d881-a480-4984-8c1f-488e29023f33) -->

Reply:
are you asking why `body` variables take preference over `:root`? that's because they are lower in the dom tree, so they override anything above it, regardless of specificity.

the layer issue is also just a dev-only thing. i'd suggest inspecting prod build to confirm everything is working
```
yarn build --filter=website
```
```
cd apps/website && yarn preview
```

Reply:
> are you asking why body variables take preference over :root? that's because they are lower in the dom tree, so they override anything above it, regardless of specificity.

`body` has a specificity of `(0, 0, 1)` and `:root` has a specificity of `(0, 1, 0)`. So I first was a bit confused regarding why `body` took preference over `:root`.

Upon looking in the inpect panel, I saw that the layer with `body` is higher list of layers in the inspect panel. But I am not sure why did the `body` layer appear higher than the `:root` layer and thus override the styles in the layer with `:root`.

> the layer issue is also just a dev-only thing. i'd suggest inspecting prod build to confirm everything is working

Sounds good, will inspect prod version.